### PR TITLE
Inline jinja2-time extension code

### PR DESCRIPTION
**Pull request by yuvaldolev**
Addresses #31

This PR has been ported over from the Cookiecutter project.
For more details see: https://github.com/cookiecutter/cookiecutter/pull/1779

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -11,9 +11,9 @@
     'Jinja2>=2.7,<4.0.0',
     'click>=7.0,<9.0.0',
     'pyyaml>=5.3.1',
-    'jinja2-time>=0.2.0',
     'python-slugify>=4.0.0',
     'requests>=2.23.0',
+    'arrow',
```
Comment (liortct) [inline setup.py]: Add versions

File: cookieninja/extensions.py [hunk 2]
```diff
@@ -1,9 +1,11 @@
 """Jinja2 extensions."""
+import arrow
```
Comment (liortct) [inline cookieninja/extensions.py]: This is not a built in package, reorder imports

File: setup.py (hunk 1 above)
Comment (tranzystorekk) [reply setup.py]: From the changelog, seems like `shift` first appears in 0.9.0. so `arrow>=0.9.0` should be fine

**Comment by tranzystorekk** [comment]
Thanks for porting this over, I should be able to package cookieninja for Void Linux once this has been released 😄

File: setup.py (hunk 1 above)
Comment (yuvaldolev) [reply setup.py]: Thanks @tranzystorek-io

Comment (yuvaldolev) [reply setup.py]: Done!

File: cookieninja/extensions.py (hunk 2 above)
Comment (yuvaldolev) [reply cookieninja/extensions.py]: Done!

**Review by liortct — APPROVED** [review]
Nice

**Pull request merged**